/*!
 * © 2020 Atypon Systems LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
export default `<html><head>
                <meta charset="UTF-8"></meta><title>Mutant p53 proteins counteract autophagic mechanism sensitizing cancer cells to mTOR
                    inhibition
                </title></head><body><div class="tmp-container"><div class="tmp-article-wrapper"><article>
    <p style="margin-bottom:0px;">Article</p><h1 style="margin-bottom:40px;margin-top:0px;" class="mainTitle">Mutant p53 proteins counteract autophagic mechanism sensitizing cancer cells to mTOR
                    inhibition
                </h1>
                <section class="article-section article-section__abstract"><h2 class="article-section__header main main">Abstract</h2><div class="article-section__content main">
                    <p>Mutations in TP53 gene play a pivotal role in tumorigenesis and cancer development. Here, we
                        report that gain‐of‐function mutant p53 proteins inhibit the autophagic pathway favoring
                        antiapoptotic effects as well as proliferation of pancreas and breast cancer cells. We found
                        that mutant p53 significantly counteracts the formation of autophagic vesicles and their fusion
                        with lysosomes throughout the repression of some key autophagy‐related proteins and enzymes as
                        BECN1 (and P‐BECN1), DRAM1, ATG12, SESN1/2 and P‐AMPK with the concomitant stimulation of mTOR
                        signaling. As a paradigm of this mechanism, we show that atg12 gene repression was mediated by
                        the recruitment of the p50 NF‐κB/mutant p53 protein complex onto the atg12 promoter. Either
                        mutant p53 or p50 NF‐κB depletion downregulates atg12 gene expression. We further correlated the
                        low expression levels of autophagic genes (atg12, becn1, sesn1, and dram1) with a reduced
                        relapse free survival (RFS) and distant metastasis free survival (DMFS) of breast cancer
                        patients carrying TP53 gene mutations conferring a prognostic value to this mutant p53‐and
                        autophagy‐related signature. Interestingly, the mutant p53‐driven mTOR stimulation sensitized
                        cancer cells to the treatment with the mTOR inhibitor everolimus. All these results reveal a
                        novel mechanism through which mutant p53 proteins promote cancer cell proliferation with the
                        concomitant inhibition of autophagy.
                    </p>
                </div></section>
                <section class="article-section article-section__abstract"><h2 class="article-section__header short short">Highlights</h2><div class="article-section__content short">
                    
                    <p>
                        <ul class="bulleted-list">
                            <li>GOF p53 mutant proteins inhibit the autophagic vesicle formation in cancer
                                cells.
                            </li>
                            <li>Mutant p53 proteins inhibit the expression of ATGs in cancer cells and patients.
                            </li>
                            <li>Mutant p53/NF‐κB p50 complex inhibits atg12 gene expression.</li>
                            <li>Mutant p53 proteins stimulate mTOR and repress AMPK signaling.</li>
                            <li>The expression of mutant p53 proteins sensitizes cancer cells to mTOR
                                inhibition.
                            </li>
                        </ul>
                    </p>
                </div></section>
            
    <section class="article-section article-section__full">
        <div class="article-section__content" id="mol220161071008-sec-sec1">
            
            <h2 class="article-section__title">1 Introduction</h2>
            <p>Mutations in the
                <i>TP53</i>
                gene occur in over 50% of human cancers, where most of them are missense mutations resulting in the
                expression of mutant forms of p53 (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0058 #mol220161071008-bib-0059">
                    Vousden and Lu, 2002; Waddell et al., 2015</a>). In addition, p53 mutated proteins acquire new
                biological properties referred to as gain‐of‐function (GOF) that contribute to the induction and
                maintenance of cancer (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0049">Santoro et al., 2014</a>). In many
                human tumors, p53 mutations are associated with high genomic instability, poor prognosis, poor response
                to chemotherapy and accelerated tumor recurrence (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0025 #mol220161071008-bib-0036 #mol220161071008-bib-0060">Ganci
                    et al., 2013; Liu et al., 2012; Walerych et al., 2012</a>). Different models have been proposed
                to explain the GOF activities of mutant p53, including binding and inactivation of the p53 family
                members p63 and p73, modulation of the activity of the transcription factors NF‐Y, E2F1, E2F4, p65
                NF‐κB, and vitamin D receptor (VDR), or the inactivation of the DNA damage sensor ataxia telangiectasia
                mutated (ATM) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0012 #mol220161071008-bib-0052 #mol220161071008-bib-0053 #mol220161071008-bib-0056 #mol220161071008-bib-0061">
                    Di Agostino et al., 2006; Stambolsky et al., 2010; Strano et al., 2007; Valenti et al., 2015; Weisz
                    et al., 2007</a>). Recently, we have documented that DNA damage with gemcitabine stabilized
                mutant p53 protein in cell nuclei triggering chemoresistance and inducing the expression of cell
                cycle‐related genes, as Cdk1 and CCNB1 increased cell growth (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0018">
                    Fiorini et al., 2015b</a>). Autophagy is a self‐eating process by which eukaryotic cells degrade
                proteins and cytoplasmic organelles through lysosomal hydrolases, thus controlling quality of the
                cytoplasm and recycling macromolecules to extract energy and to overcome nutritional or
                microenvironmental stressful situations (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0057">Viry et al., 2014</a>
                ). Several studies indicate that wild type p53 triggers the autophagic machinery in human cancer cells
                involving many pathways as the stimulation of the nutrient energy sensor AMP‐activated protein kinase
                (AMPK), the inhibition of the mammalian target of rapamycin (mTOR) by up‐regulation of PTEN and TSC1,
                and the induction of the autophagy‐related gene DRAM1 (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0008 #mol220161071008-bib-0009 #mol220161071008-bib-0015 #mol220161071008-bib-0028">
                    Comel et al., 2014; Crighton et al., 2007; Drakos et al., 2009; Gomes et al., 2015</a>).
                Interestingly, autophagy regulation is strictly interconnected with the aberrant setting of cancer cell
                metabolism as revealed by the fact that mTOR and AMPK pathways are both the master regulators of
                autophagy and the most important sensors of the cellular energy status (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0013">Dodson et al., 2013</a>). In particular, mTOR complex
                stimulates anabolic biosynthesis for cancer cell growth and inhibits autophagy, while AMPK signaling
                triggers degradation of macromolecules, including lysosomal autophagic catabolism (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0010">DeBerardinis and Thompson, 2012</a>). Recently, Zhou et al.
                have shown that GOF mutant p53 inhibited AMPK signaling in head and neck cancer cells directly binding
                to the AMPKα subunit thus stimulating anabolic growth and gaining its oncogenic function (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0062">Zhou et al., 2014</a>). Other studies demonstrated the
                involvement of mutant p53 in the regulation of cancer metabolism, throughout the transcriptional
                modulation of SREBP1, a downstream target of AMPK (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0020">Freed‐Pastor
                    et al., 2012</a>). However, despite a study describing a correlation between the subcellular
                localization of overexpressed p53 mutant proteins and the regulation of autophagy (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0043">Morselli et al., 2008</a>), the functional involvement of
                endogenous mutant p53 in the regulation of autophagy in cancer cells and the identification of the
                associated molecular mechanisms remain largely unknown.
            </p>
            <p>In the present work, we ascertain that GOF mutant p53 proteins are able to orchestrate a plethora of
                events addressed to counteract autophagy in the various phases of the process, thus contributing to
                inhibit apoptosis and to sustain oncogenic activity. Notably, we document that mutant p53 interacts with
                the p50 NF‐κB subunit onto the
                <i>atg12</i>
                promoter inhibiting its expression and resulting in the repression of the autophagic machinery.
                Furthermore, we point out that cancer cells bearing endogenous mutant p53 are more sensitive to mTOR
                inhibition than cancer cells expressing wild type p53 protein, providing a conceivable personalized
                therapy for cancer patients carrying GOF mutations of
                <i>TP53</i>
                gene.
            </p>
        </div>
        <div class="article-section__content" id="mol220161071008-sec-sec2">
            
            <h2 class="article-section__title">2 Material and methods</h2>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.1">
                
                <h3 class="article-section__sub-title">2.1 Chemicals</h3>
                <p>Chloroquine diphosphate (CQ), 3‐methyladenine (3MA), and everolimus (EVE; RAD‐001) were obtained from
                    Sigma (Milan, Italy). Compound C (CC) solution was obtained from Calbiochem (Merck; Darmstadt,
                    Germany).
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.2">
                
                <h3 class="article-section__sub-title">2.2 Cell culture</h3>
                <p>PaCa3 (WTp53), Panc1 (mutant p53‐R273H), PaCa44 (mutant p53‐C176S), MiaPaCa2 (mutant p53‐R248W),
                    Suit‐2 (mutant p53‐R273H) and AsPC1 (p53‐null) human pancreatic adenocarcinoma cell lines were grown
                    in RPMI medium (Life Technologies, Milan, Italy), while lung cancer H1299 (p53‐null), breast cancer
                    MCF7 (WTp53), SKBr3 (mutant p53‐R175H), MDA‐MB‐468 (mutant p53‐R273H) and MDA‐MB‐231 (mutant
                    p53‐R280K) cell lines were cultured in DMEM medium (Life Technologies, Milan, Italy). Both culture
                    media were supplemented with 10% FBS, and 50 μg/ml gentamicin sulfate (BioWhittaker, Lonza, Bergamo,
                    Italy). Cell lines were incubated at 37 °C with 5% CO<sub>2</sub>. The list of the cell lines used
                    in this study and their p53 status are summarized in<a class="tableLink" title="Link to table" href="#mol220161071008-tbl-0001">Table
                        1</a>.
                </p>
                <header class="article-table-caption"><span class="table-caption__label">
                Table 1.
            </span>Tissue of origin and p53 status of cancer cell lines.</header><div class="article-table-content" id="mol220161071008-tbl-0001">
                    
                    
                        <div class="article-table-content-wrapper"><table class="table article-section__table">
                            
                            
                            
                            
                            <thead>
                                <tr>
                                    <th class="bottom-bordered-cell center-aligned">Cancer cell lines</th>
                                    <th class="bottom-bordered-cell center-aligned">Tissue origin</th>
                                    <th class="bottom-bordered-cell center-aligned">p53 status</th>
                                    <th class="bottom-bordered-cell center-aligned">Mutation</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <td class="center-aligned">PaCa3</td>
                                    <td class="center-aligned">Pancreas</td>
                                    <td class="center-aligned">Wild‐type</td>
                                    <td class="center-aligned">None</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">MCF7</td>
                                    <td class="center-aligned">Breast</td>
                                    <td class="center-aligned">Wild‐type</td>
                                    <td class="center-aligned">None</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">MiaPaca2</td>
                                    <td class="center-aligned">Pancreas</td>
                                    <td class="center-aligned">Mutated</td>
                                    <td class="center-aligned">R248W</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">PaCa44</td>
                                    <td class="center-aligned">Pancreas</td>
                                    <td class="center-aligned">Mutated</td>
                                    <td class="center-aligned">C176S</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">Panc1</td>
                                    <td class="center-aligned">Pancreas</td>
                                    <td class="center-aligned">Mutated</td>
                                    <td class="center-aligned">R273H</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">Suit‐2</td>
                                    <td class="center-aligned">Pancreas</td>
                                    <td class="center-aligned">Mutated</td>
                                    <td class="center-aligned">R273H</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">MDA‐MB‐468</td>
                                    <td class="center-aligned">Breast</td>
                                    <td class="center-aligned">Mutated</td>
                                    <td class="center-aligned">R273H</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">SkBr3</td>
                                    <td class="center-aligned">Breast</td>
                                    <td class="center-aligned">Mutated</td>
                                    <td class="center-aligned">R175H</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">MDA‐MB‐231</td>
                                    <td class="center-aligned">Breast</td>
                                    <td class="center-aligned">Mutated</td>
                                    <td class="center-aligned">R280K</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">AsPC1</td>
                                    <td class="center-aligned">Pancreas</td>
                                    <td class="center-aligned">Null</td>
                                    <td class="center-aligned">Gene deleted</td>
                                </tr>
                                <tr>
                                    <td class="center-aligned">H1299</td>
                                    <td class="center-aligned">Lung</td>
                                    <td class="center-aligned">Null</td>
                                    <td class="center-aligned">Gene deleted</td>
                                </tr>
                            </tbody>
                        </table></div>
                    
                </div>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.3">
                
                <h3 class="article-section__sub-title">2.3 Cell proliferation assay</h3>
                <p>Cells were seeded in 96‐well plates (5 × 10<sup>3</sup> cells/well) and the day after were incubated
                    with various compounds at the indicated conditions or transfected with the indicated constructs (see
                    figure legends). At the end of the treatments, cell growth was measured by Crystal Violet assay
                    (Sigma, Milan, Italy) according to the manufacturer's protocol, and absorbance was measured by
                    spectrophotometric analysis (A<sub>595</sub>nm).
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.4">
                
                <h3 class="article-section__sub-title">2.4 Transient transfection assays</h3>
                <p>Exponentially growing cells were seeded at 5 × 10<sup>3</sup> cells/well in 96‐well plates or at
                    2.5 × 10<sup>5</sup> cells in 60 mm cell culture plates. The ectopic expression of mutant p53 in
                    p53‐null cancer cells was carried out transfecting pcDNA3‐mutp53R273H or pcDNA3‐mutp53R175H
                    expression vectors, or their relative negative control (pcDNA3). The ectopic expression of wild type
                    or dominant negative (DN)‐AMPK subunit γ2 was obtained by transfection of the pcDNA5/FRT expression
                    vector containing the human wild type AMPK γ2 subunit or the mutated R531G protein which were kindly
                    provided by Dr. Hawley (University of Dundee, Scotland, UK). Wild‐type and mutant p53 protein
                    expression was transiently knocked‐down by transfection with pRSUPER‐p53 or pLVTHM‐p53 vectors or
                    their negative controls (pRSUPER or pLVTHM), kindly provided by Dr. Agami (The Netherlands Cancer
                    Institute, Amsterdam) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0004">Brummelkamp et al., 2002</a>) and
                    by Dr. Sergio Ruiz (CNIO, Madrid, Spain), respectively. Commercial siRNA smart pool of three
                    oligonucleotides (si‐p53) transiently targeting p53 (Santa Cruz Biotech. sc‐29435) was used in the
                    experiments of RT‐qPCR to exclude back‐side effects and to confirm the robustness of the data. A
                    si‐GFP as non‐silencing control 5′‐GGCTACGTCCAGGAGCGCACC‐3′ was used as a negative control. The
                    silencing transfections were carried out for 48 h using Lipofectamine 2000 (Life Technologies),
                    according to the manufacturer's instructions. Knockdown of Atg5 expression was performed by
                    shRNA‐Atg5 cloned in pMSCV‐Puro‐miR30 vector (or its negative empty vector) kindly provided by Dr.
                    Hans‐Uwe Simon (University of Bern, Switzerland) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0038">Maskey
                        et al., 2013</a>). Knock‐down of Beclin1 expression was obtained by transfecting cells with
                    specific Beclin1 small interfering (si) RNA having the following sequences:
                    5′‐ACAGUGAAUUUAAACGACAGCAGCU‐3′ and 5′‐AGCUGCUGUCGUUUAAAUUCACUGU‐3′ and with a siRNA‐CTRL (negative
                    control): 5′‐CAGUCGCGUUUGCGACUGG‐3′ purchased from Life Technologies. Knock‐down of p50 NF‐κB
                    subunit expression was obtained by transfecting cells with a specific 21‐nt‐long interfering RNA
                    duplex with two 3'‐end overhang dT nucleotides. The sequence of the antisense strand of the anti‐p50
                    siRNA was 5′‐AGUCCAGGAUUAUAGCCCCdTdT‐3' (MWG Biotec, Ebersberg, Germany; or Dharmacon, Lafayette,
                    CO) as reported (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0034">Laderach et al., 2003</a>). Cells were
                    transfected by siRNAs at a final concentration of 50 nM using Lipofectamine 2000 (Life
                    Technologies), according to the manufacturer's instructions.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.5">
                
                <h3 class="article-section__sub-title">2.5 RNA isolation and quantitative real‐time PCR analysis</h3>
                <p>Total RNA was extracted from cells using TRIZOL Reagent (Life Technologies) in accordance with the
                    manufacturer's instructions. Five micrograms of total RNA were reverse‐transcribed at 37 °C for
                    60 min in the presence of random hexamers and Moloney murine leukemia virus reverse transcriptase
                    (Life Technologies). Transcripts were measured by real‐time PCR using the SYBR Green assay (Applied
                    Biosystems, Carlsbad, CA, USA) with a StepOne instrument (Applied Biosystems). PCR analysis was
                    carried out using specific oligonucleotides for the genes listed in<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Table 1</a>. The primers were designed
                    with Primer3 version 0.4.0 (<a title="Link to external resource" href="http://frodo.wi.mit.edu/primer3/">
                        http://frodo.wi.mit.edu/primer3/</a>). All primer sets worked under identical quantitative PCR
                    cycling conditions with similar efficiencies to obtain simultaneous amplification in the same run.
                    The 2
                    <sup>−ΔΔCT</sup>
                    method for relative quantification of gene expression was used to determine mRNA expression levels.
                    <i>GAPDH</i>
                    gene expression was used as endogenous control to standardize mRNA expression. All reactions were
                    performed in duplicate from three independent experiments.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.6">
                
                <h3 class="article-section__sub-title">2.6 ChIP experiments</h3>
                <p>1% formaldehyde cross‐linking and chromatin immunoprecipitations were performed as described in (
                    <a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0012 #mol220161071008-bib-0055">Di Agostino et al., 2006; Valenti
                        et al., 2011</a>). The chromatin solution was immunoprecipitated with sheep anti‐p53 Ab7
                    (Millipore, Billerica, MA, USA), rabbit anti‐p50 (Santa Cruz Biotech. SC‐7178), rabbit anti‐p65
                    (Santa Cruz Biotech. SC‐372), or no antibody as negative control. The immunoprecipitations were
                    performed using Pierce ChIP‐grade Protein A/G magnetic beads (Thermo Fisher Scientific, Rockford,
                    IL, USA). Primers used for the amplification of the different regulatory regions are listed in<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Table 2</a>. NF‐κB p50 Region 2
                    consensus sequence on ATG12 promoter was selected from the literature and NF‐κB p50 Region 1, the
                    consensus sequence was identified by MatInspector software (<a title="Link to external resource" href="http://www.genomatix.de">
                        www.genomatix.de</a>). The promoter occupancy was analyzed by RT‐qPCR using the Fast SYBR
                    Green assay (Applied Biosystems, Carlsbad, CA, USA) and the StepOne™ Real‐Time PCR System (Applied
                    Biosystems). Normalization was performed to the amount of input chromatin. The ChIP results were
                    further normalized on the qPCR of a H1H2B2, the genic region resulted negative for the recruitment
                    of mutant p53 (<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 1</a>) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0003">Biagioni et al., 2012</a>).
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.7">
                
                <h3 class="article-section__sub-title">2.7 Immunoblot analysis</h3>
                <p>Cells were harvested, washed in PBS, and re‐suspended in lysis buffer in the presence of phosphatase
                    and protease inhibitors (50 mM Tris–HCl pH 8, 150 mM NaCl, 1% Igepal CA‐630, 0.5% Na‐Doc, 0.1% SDS,
                    1 mM Na<sub>3</sub>VO<sub>4</sub>, 1 mM NaF, 2.5 mM EDTA, 1 mM PMSF, and 1× protease inhibitor
                    cocktail). After incubation on ice for 30 min, the lysates were centrifuged at 14,000 × g for
                    10 min at 4 °C and the supernatant fractions were used for Western blot analysis. Protein
                    concentration was measured by Bradford reagent (Pierce, Milan, Italy) using bovine serum albumin as
                    a standard. Protein extracts (50 μg/lane) were resolved on a 12% SDS‐polyacrylamide gel and
                    electro‐blotted onto PVDF membranes (Millipore, Milan, Italy). Membranes were blocked in 5% low‐fat
                    milk in TBST (50 mM Tris pH 7.5, 0.9% NaCl, 0.1% Tween 20) for 1 h at room temperature and probed
                    overnight at 4 °C with a mouse polyclonal anti‐p53 (1:500) (Santa Cruz, #sc‐263), rabbit monoclonal
                    anti‐LC3 (1:1000) (Cell Signaling, #2775), rabbit polyclonal anti‐phospho(Ser15)Beclin1 (1:1000)
                    (Cell Signaling #13825) rabbit polyclonal anti‐Beclin1 (1:1000) (GeneTex, #GTX113039), rabbit
                    monoclonal glyceraldehyde 3‐phosphate dehydrogenase (GAPDH) (1:1000) (Cell Signaling, #5174S),
                    rabbit monoclonal anti‐phospho (Thr172)AMPK (1:1000) (Cell Signaling #2535), rabbit monoclonal
                    anti‐AMPK (1:1000) (Cell Signaling #2603), mouse monoclonal anti‐phospho (Thr389)p70S6K (1:1000)
                    (Cell Signaling #9206), rabbit monoclonal anti‐p70S6K (1:1000) (Cell Signaling #2708), rabbit
                    polyclonal anti‐SESN1 (1:1000) (GeneTex #GTX116926), rabbit polyclonal anti‐SESN2 (1:1000) (GeneTex
                    #118141), rabbit polyclonal anti‐NF‐κB p50 (1:1000) (Santa Cruz #sc‐7178), rabbit polyclonal
                    anti‐NF‐κB p65 (1:1000) (Santa Cruz #sc‐372), mouse monoclonal anti‐alpha‐tubulin (1:2500) (Oncogene
                    #CP06), or rabbit monoclonal anti‐phospho (Tyr705)STAT3 (1:1000) (Cell Signaling #9131) antibodies.
                </p>
                <p>Horseradish peroxidase conjugated anti‐mouse or anti‐rabbit IgGs (1:8000 in blocking solution)
                    (Upstate Biotechnology, Milan, Italy) were used as secondary antibodies. Immunodetection was carried
                    out using chemiluminescent substrates (Amersham Pharmacia Biotech, Milan, Italy) and recorded using
                    a HyperfilmECL (Amersham Pharmacia Biotech). ECL results were scanned and the amount of each protein
                    band quantified using NIH Image J software (<a title="Link to external resource" href="http://rsb.info.nih.gov/nih-image/">
                        http://rsb.info.nih.gov/nih‐image/</a>).
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.8">
                
                <h3 class="article-section__sub-title">2.8 Immunoprecipitation assay</h3>
                <p>Cell extracts were solubilized in lysis buffer with 150 mM Hepes pH 7.5, 300 mM NaCl, 1% Triton‐X100,
                    phosphatase and protease inhibitors. Cells were harvested and lysed by sonication in lysis buffer
                    and cleared by centrifugation. Protein concentrations were determined by colorimetric assay
                    (Bio‐Rad, Hercules, CA, USA). For each immunoprecipitation, 1 μg of antibody and 1 μg of rabbit or
                    sheep IgG (Santa Cruz Biotech) were used as controls. To immunoprecipitate, we used rabbit
                    polyclonal anti‐p50 (sc‐7178) from Santa Cruz Biotech and sheep polyclonal anti‐p53 Ab7 (PC35‐1EA)
                    from Millipore. 1 mg of pre‐cleared extracts were diluted in lysis buffer containing 0.05% BSA and
                    incubated with protein A/G‐Agarose beads (Thermo Fisher Scientific, Rockford, IL, USA) and
                    antibodies, under constant shaking at 4 °C for 3 h. Bead‐bound immunocomplexes were rinsed with
                    lysis buffer and eluted in 50 μl of SDS sample buffer for Western blotting. Western blotting was
                    performed using the following primary antibodies: mouse monoclonal p53 (DO1), GAPDH (Santa Cruz
                    Biotech), rabbit polyclonal anti‐NF‐κB p65 (1:1000) (Santa Cruz #sc‐372), rabbit polyclonal anti‐p50
                    (sc‐7178) and actin (Sigma) antibodies. Western blot analysis was performed with the aid of the
                    enhanced chemiluminescence system (Thermo Fisher Scientific, Rockford, IL, USA) and the acquisition
                    of the images by Uvitec technology (Eppendorf).
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.9">
                
                <h3 class="article-section__sub-title">2.9 Apoptosis assay</h3>
                <p>Cells were seeded in 96‐well plates (5× 10<sup>3</sup> cells/well). Twenty‐four hours later, cells
                    were treated as indicated in the figure legends. At the end of the treatment, cells were fixed with
                    2% paraformaldehyde in PBS for 30 min at room temperature, washed twice with PBS and stained with
                    annexinV/FITC (Bender MedSystem, Milan, Italy) in binding buffer (10 mM HEPES/NaOH pH 7.4, 140 mM
                    NaCl, and 2.5 mM CaCl<sub>2</sub>) for 10 min at room temperature in the dark. Cells were then
                    washed with binding buffer and fluorescence was measured using a multimode plate reader (Ex
                    <sub>485nm</sub>
                    and Em<sub>535nm</sub>) (GENios Pro, Tecan, Milan, Italy). The values were normalized on cell
                    proliferation by Crystal Violet assay.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.10">
                
                <h3 class="article-section__sub-title">2.10 Monodansylcadaverine staining and autophagosome formation assay</h3>
                <p>To quantify the induction of autophagy, cells were incubated with the fluorescent probe
                    monodansylcadaverine (MDC; Sigma, Milan, Italy), accordingly with the guidelines for studying
                    autophagy (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0033">Klionsky et al., 2016</a>). MDC is a selective
                    marker for acidic vesicular organelles, such as autophagic vacuoles. Briefly, cells were seeded in
                    96‐well plates (5 × 10<sup>3</sup> cells/well) and treated with various compounds as indicated in
                    the figure legends. At the end of the treatments, cells were incubated in culture medium containing
                    50 μM MDC at 37 °C for 15 min. Cells were then washed with Hanks buffer (20 mM Hepes pH 7.2, 10 mM
                    glucose, 118 mM NaCl, 4.6 mM KCl, and 1 mM CaCl<sub>2</sub>) and fluorescence was measured using a
                    multimode plate reader (Ex
                    <sub>340nm</sub>
                    and Em<sub>535nm</sub>) (GENios Pro, Tecan, Milan, Italy). The values were normalized for cell
                    proliferation by Crystal Violet assay.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.11">
                
                <h3 class="article-section__sub-title">2.11 Acridine orange staining and autolysosome detection</h3>
                <p>Cells were seeded in a 24‐well plate on glass cover‐slips at a density of 6 × 10<sup>4</sup>/well.
                    After 24 h, cells were transfected with pRSUPER‐p53 or with the empty pRSUPER vector by using
                    Lipofectamine 2000 according to the manufacturer's instructions. 48 h after transfection cells were
                    rinsed in phosphate buffer saline (PBS) and stained with 6 μg/ml acridine orange (AO). Cell images
                    were captured using a fluorescence microscope Leica DM6000 (Leica Microsystem, Manheim, Germany) at
                    40× magnification and processed using Adobe Photoshop and NIH Image J software.
                </p>
                <p>To quantify autolysosome formation, cells transfected with pRSUPER‐p53 vector or its negative control
                    (pRSUPER) were stained with AO following the manufacturer's instructions. Quantification of
                    autolysosomes was performed measuring the red/green fluorescence intensity ratio of AO staining (AO
                    green fluorescence Ex
                    <sub>485nm</sub>
                    and Em<sub>535nm</sub>; AO red fluorescence Ex
                    <sub>430nm</sub>
                    and Em<sub>590nm</sub>) with a multimode plate reader (GENios Pro, Tecan, Milan, Italy). Values were
                    normalized on cell proliferation by crystal violet assay.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.12">
                
                <h3 class="article-section__sub-title">2.12 LC3‐GFP and lysosome co‐localization analysis</h3>
                <p>Cells were seeded in a 24‐well plate on glass cover‐slips at a density of 6 × 10<sup>4</sup>/well.
                    After 24 h, cells were co‐transfected with pEGFP‐LC3B and pRSUPER‐p53 vector or with pEGFP‐LC3B and
                    the pRSUPER empty vector by using Lipofectamine 2000, according to the manufacturer's instructions.
                    48 h after transfection, lysosomes were stained with 50 nM Lysotracker‐RED‐DN99 (Life Technologies)
                    and nuclei were stained with 0.1 μg/ml Hoechst (Life Technologies). Cells were than rinsed in PBS
                    and fixed in 4% (w/v) paraformaldehyde. Cover‐slips were mounted over slides in AF1 medium (Dako).
                    Cell images were captured using a confocal laser‐scanning fluorescence microscope Leica SP5 (Leica
                    Microsystem, Manheim, Germany) at 63× magnification and processed using Adobe Photoshop and NIH
                    Image J software.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.13">
                
                <h3 class="article-section__sub-title">2.13 mRNA expression data</h3>
                <p>We used the publicly available data sets: GSE22358 (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0027">Gluck
                    et al., 2012</a>) and GSE31812 (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0020">Freed‐Pastor et al.,
                    2012</a>). The data sets were analyzed by the Oncomine platform
                    <a title="Link to external resource" href="http://www.oncomine.org">www.oncomine.org</a>
                    (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0046">Rhodes et al., 2004</a>). The mRNA expression data were
                    measured using Agilent UNC Perou Lab Homo sapiens 1 × 44K Custom Array and Human Gene 1.0 ST arrays,
                    respectively.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.14">
                
                <h3 class="article-section__sub-title">2.14 Kaplan‐Meier analysis</h3>
                <p>Gene expression dataset raw data and survival information of 108 BCL patients with
                    <i>TP53</i>
                    mutated were downloaded from GEO (<a title="Link to external resource" href="http://www.ncbi.nlm.nih.gov/geo/">
                        http://www.ncbi.nlm.nih.gov/geo/</a>) and the Kaplan‐Meier analysis was performed by using the
                    on‐line tool
                    <a title="Link to external resource" href="http://www.kmplot.com">www.kmplot.com</a>
                    (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0031">Gyorffy et al., 2010</a>) where eight public datasets
                    were considered. Gene expression profiles were performed by using Affymetrix HG‐U133A (GPL96) and
                    HG‐U133 Plus 2.0 (GPL570) microarrays were only considered. The latter are the most frequently used
                    and have 22,277 probe sets in common. To analyze the prognostic value of genes, the cohorts were
                    divided into two groups according to the median (or upper/lower quartile) expression of gene
                    signature considered. The two groups were compared in terms of relapse free survival (RFS) and
                    distant metastasis free survival (DMFS). A survival curve was displayed, and the hazard ratio with
                    95% confidence intervals and logrank P value are calculated. P values &lt; 0.05 were considered
                    statistically significant.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec2.15">
                
                <h3 class="article-section__sub-title">2.15 Statistical analysis</h3>
                <p>ANOVA (post hoc Bonferroni) analysis was performed by GraphPad Prism 5 software. Student's t‐test
                    (two‐tailed) was also conducted. P value &lt; 0.05 was indicated as statistically significant.
                    Values are the means of three independent experiments (±SD).
                </p>
            </div>
        </div>
        <div class="article-section__content" id="mol220161071008-sec-sec3">
            
            <h2 class="article-section__title">3 Results</h2>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec3.1">
                
                <h3 class="article-section__sub-title">3.1 Gain of function p53 mutant proteins inhibit the autophagic vesicles formation in
                    cancer cells
                </h3>
                <p>To study the functional role of GOF mutant p53 proteins in the autophagic mechanism, we first
                    analyzed the endogenous level of autophagosome vesicle formation by staining diverse cancer cell
                    lines with the fluorescent probe MDC. Cancer cells with different missense mutations of
                    <i>TP53</i>
                    gene had the endogenous level of autophagosome vesicles about 5–6 folds lower than cells with wild
                    type
                    <i>TP53</i>
                    alleles (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>A). When PaCa3 and MCF7 cell lines
                    (expressing a wild type p53 protein) were knocked down for p53 expression, the autophagosome
                    formation was inhibited, accordingly with the literature (<a class="figureLink" href="#mol220161071008-fig-0001">
                        Figure 1</a>B and C) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0009">Crighton et al., 2007</a>).
                    Conversely, the autophagosome formation was significantly increased after knock‐down of GOF mutant
                    p53 in six cancer cell lines (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>B and C).
                    Consistent with this, overexpression of mutant p53 R175H or R273H proteins in AsPC1 cells (null for
                    p53 expression) produced a drastic reduction of MDC probe incorporation revealing an inhibitory role
                    of mutant p53 of the autophagic process (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>B and
                    C). To further support the idea that mutant p53 may have a role in the inhibition of the autophagic
                    event, we analyzed the expression of the autophagic receptor/adaptor SQSTM1/p62 (sequestosome 1),
                    whose massive accumulation is usually a consequence of autophagy impairment and accumulation of
                    aggregated structures positive for ubiquitin (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0032">Katsuragi
                        et al., 2015</a>). Accordingly, we observed that mutant p53 knockdown triggered p62
                    expression decrease (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>D). Furthermore, we
                    evaluated the expression levels of the lipidated and truncated isoform II of light chain 3 protein
                    (LC3‐II), which is functionally involved in the formation and maturation of autophagic vesicles. Our
                    findings showed that LC3‐II protein expression levels considerably decreased in AsPC1 cells after
                    ectopic expression of mutant R175H or R273H p53 proteins while increased in mutant p53‐knocked down
                    cells (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>D). We also observed that chloroquine
                    (CQ) further increased the amount of LC‐II in silencing mutant p53 conditions (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>D), accordingly with the hypothesis of the
                    autophagic inhibitory flux inside the blockage of the lysosomal degradation. According to this,
                    confocal microscopy analysis revealed that mutant p53 counteracts the fusion between autophagic
                    vesicles (identified by LC3‐GFP overexpression) and lysosomes (identified by lysotracker red probe)
                    since the merged image of Panc1‐shp53 cells showed a significant co‐localization of LC3‐II with
                    lysosomes represented by yellow puncta and confirmed by the RGB profile (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>E). In<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 2</a>, we included a negative
                    control about formation of autophagic vesicles after knock‐down of mutant p53 in the absence or
                    presence of the autophagy inhibitor 3‐methyladenine. Furthermore, we have examined the formation of
                    autolysosomes (autophagic vesicles joined with lysosomes) using the fluorescent probe acridine
                    orange (AO), which changes its fluorescence emission from green to red upon accumulation into
                    lysosomal acidic compartments. The typical pH acidification of lysosomes during autophagic
                    stimulation has been examined by AO probe staining in knocked down mutant p53 conditions (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>F and G).<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>F showed that mutant p53 cell lines (Panc1
                    and SKBr3) had a high red/green fluorescence ratio in the sh‐p53 condition, conversely wild type
                    p53 cells (MCF7) showed a decrease of the autolysosome formation (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>F). The microscopy experiments confirmed
                    these results revealing the presence of red areas (acidic lysosomes containing accumulation of AO
                    probe) into the cytoplasm of the cells interfered for mutant p53 expression (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>G). Overall these data strongly support the
                    hypothesis that GOF mutant p53 proteins may contribute to prevent both the formation of the
                    autophagic vesicles and their fusion with lysosomes in cancer cells.
                </p>
                <section class="article-section__inline-figure"><figure class="figure" id="mol220161071008-fig-0001"><img class="figure__image" src="https://placeholdit.imgix.net/~text?txtsize=33&amp;txt=TestImage&amp;w=500&amp;h=500"></img><figcaption class="figure__caption"><strong>Figure
                1</strong> 
                        Mutant p53 counteracts autophagy. (A) The endogenous basal level of autophagosomes in cancer
                        cells carrying WT or mutant p53 was analyzed using the incorporation of the fluorescent probe
                        monodansylcadaverine (MDC). Cells were seeded in 96‐well plates, incubated overnight, and
                        stained with MDC as described in Material and Methods. (B) Cells were transfected with the
                        pRSuper‐p53 vector (shp53) to knockdown p53 expression (see Material and Methods) or with
                        plasmids coding mutp53R175H, mutp53R273H proteins, or their relative negative controls (shCTRL
                        or mock, respectively). Autophagosome formation assay was performed using the incorporation of
                        MDC probe. (C) Western blot of p53 using the protein extract of the samples used for (B) to test
                        the effective knock down of WT or mutant p53 and the overexpression of mutant p53 in the various
                        cell lines indicated. (D) Panc1 cells were transfected with pRSuper‐p53 vector (shp53) or its
                        negative control (shCTRL) and AsPC‐1 cells with plasmids for mutant p53 overexpression or its
                        mock vector. Panc1 cells were pre‐treated with 5 μM chloroquine (CQ) for 1 h before cell
                        transfection. Whole‐cell extracts were used for Western blot analysis of the autophagic proteins
                        p62/SQSTM1 and LC3 (isoforms I and II), p53 and GAPDH (as control loading). (E) 48 h after
                        co‐transfection with plasmids coding for LC3‐GFP and pRSuper‐p53 vector (shp53) or LC3‐GFP and
                        the empty vector (negative control; CTRL), Panc1 cells were fixed. Lysosomes and nuclei were
                        stained with Lysotracker (red) and Hoescht (blue), respectively. The RGB profile plotted along
                        the dashed line drawn in the merge image is also shown. Merge and single channel images come
                        from a single z‐plane. Scale bar 10 μm. (F) Autolysosome formation analyzed by red/green
                        fluorescence intensity ratio quantification of acridine orange (AO) staining in the indicated
                        cells transfected with pRSuper‐p53 vector (shp53) or with its negative control (shCTRL). (G)
                        48 h after the p53 depletion Panc1 cells were stained with an AO solution (1:1500 in PBS) and
                        observed at 40× magnification. Scale bar 40 μm. All the experiments presented in this figure are
                        representative of three biological replicates. P‐values were calculated with two‐tailed t‐test.
                        Statistical analysis: *p &lt; 0.05 shp53 vs shCTRL; §p &lt; 0.05 R175H or R273H vs mock.
                    </figcaption></figure></section>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec3.2">
                
                <h3 class="article-section__sub-title">3.2 Autophagy inhibition by mutant p53 increases the proliferation of cancer cells
                </h3>
                <p>Mutant p53 proteins have been implicated in the induction of cell proliferation, chemoresistance,
                    disruption of tissue architecture, promotion of migration, invasion and metastasis, and several
                    other pro‐oncogenic properties (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0001 #mol220161071008-bib-0012 #mol220161071008-bib-0021 #mol220161071008-bib-0026 #mol220161071008-bib-0060">
                        Adorno et al., 2009; Di Agostino et al., 2006; Freed‐Pastor and Prives, 2012; Girardini et al.,
                        2011; Walerych et al., 2012</a>). In this study, we considered cancer cell lines where the
                    role of tumor suppressor wild type p53 and oncogenic mutant p53 in the regulation of cell
                    proliferation was well assessed (<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary
                        Figure 3</a>) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0006 #mol220161071008-bib-0035 #mol220161071008-bib-0044">Cadwell
                        and Zambetti, 2001; Levine, 1997; Muller and Vousden, 2013</a>). To unravel the possible
                    biological role of autophagy inhibition in the aberrant proliferation induced by mutant p53, we
                    knocked‐down the essential autophagic protein Atg5 in Panc1, MiaPaCa2, and MDA‐MB‐468 cells (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0041">Mizushima et al., 1998</a>). The MDC assay established
                    that Atg5 knockdown effectively repressed the autophagosome formation triggered by mutant p53
                    downregulation (<a class="figureLink" href="#mol220161071008-fig-0002">Figure 2</a>A;
                    <a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 4</a>
                    <b>)</b>. Importantly, the depletion of Atg5 caused a reversion of cell growth inhibition (<a class="figureLink" href="#mol220161071008-fig-0002">Figure 2</a>B) and a decrease of the apoptosis (<a class="figureLink" href="#mol220161071008-fig-0002">Figure 2</a>C) stimulated by mutant p53 knockdown. These
                    findings indicated that autophagy inhibition strongly contributed to the increased proliferation and
                    to the concomitant down‐regulation of the apoptotic stimuli driven by mutant p53. Similar results
                    are also observed using 3‐methyladenine (3MA) as autophagy inhibitor at non‐toxic concentrations (
                    <a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 2</a>).
                </p>
                <section class="article-section__inline-figure"><figure class="figure" id="mol220161071008-fig-0002"><img class="figure__image" src="https://placeholdit.imgix.net/~text?txtsize=33&amp;txt=TestImage&amp;w=500&amp;h=500"></img><figcaption class="figure__caption"><strong>Figure
                2</strong> 
                        Impact of autophagy inhibition by mutant p53 on cell growth and apoptosis. Panc1, MiaPaCa2 and
                        MDA‐MB‐468 cell lines were seeded in 96‐well plates and transfected with pRSuper‐p53 vector
                        (shp53) and/or pMSCV‐Puro‐miR30‐Atg5 (shAtg5), or their relative negative controls (shCTRL) for
                        48 h. Autophagosome formation (A), cell growth (B), and apoptosis (C) were determined using MDC
                        assay, crystal violet colorimetric assay and annexinV/FITC binding assay, respectively. All the
                        experiments presented in this Figure are representative of three biological replicates. P‐values
                        were calculated with two‐tailed t‐test. Statistical analysis: *p &lt; 0.05 shp53 vs shCTRL; §p &lt; 0.05
                        shp53+shATG5 vs shp53.
                    </figcaption></figure></section>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec3.3">
                
                <h3 class="article-section__sub-title">3.3 Mutant p53 protein inhibits the expression of autophagic genes in tumoral cell lines
                    and cancer patients
                </h3>
                <p>Autophagic pathway is a complex sequence of biological events leading to formation, maturation and
                    fusion of autophagosomes with lysosomes to allow the degradation and recycling of cellular
                    components (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0057">Viry et al., 2014</a>). These events are
                    driven by a number of autophagy‐related genes (ATGs), which are mostly transcriptionally induced by
                    autophagic stimuli such as nutritional deprivation, infections or metabolic and oncogenic stress (
                    <a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0023">Galluzzi et al., 2015</a>). In the present study, we have
                    shown that tumor cells carrying mutant p53 proteins escaped autophagic activity (<a class="figureLink" href="#mol220161071008-fig-0001 #mol220161071008-fig-0002">Figures 1 and 2</a>). In order
                    to investigate the role of mutant p53 in autophagy inhibition, we studied whether mutant p53
                    proteins could control the expression of some crucial ATGs, as<i>atg12</i>,<i>becn1</i>,<i>sesn1</i>
                    ,
                    <i>sesn2</i>
                    and
                    <i>dram1</i>
                    genes. The depletion of mutant p53 by p53 siRNA smart pool oligos transfection (sip53), determined
                    an increase of the mRNA expression in most of these ATGs in Panc1, MDA‐MB‐468 and SKBr3 cell lines (
                    <a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>A,<a class="suppLink" href="#mol220161071008-sup-appsec1">
                        Supplementary Figure 5</a>). In these cell lines, we have checked the CCNB1 mRNA expression
                    as a control of our cellular systems (<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary
                        Figure 6</a>). Cyclin B protein is related to the cell cycle proteins, important for the
                    control of the cell cycle progression, and is a well‐established mutant p53 target gene (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0011 #mol220161071008-bib-0012">Di Agostino et al., 2015; Di
                        Agostino et al., 2006</a>). Conversely, the depletion of WTp53 expression in MCF7 breast
                    cancer cell line led to a general decrease of the expression of ATGs (<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>A,<a class="suppLink" href="#mol220161071008-sup-appsec1">
                        Supplementary Figure 5</a>). Accordingly, ectopic expression of mutp53R175H or mutp53R273H in
                    p53‐null H1299 cells led to a significant reduction of ATGs transcripts (<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>A,<a class="suppLink" href="#mol220161071008-sup-appsec1">
                        Supplementary Figure 5</a>,<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary
                        Figure 6</a>). To further investigate the inverse relationship between the expression of
                    mutant p53 and two representative ATGs (
                    <i>atg12</i>
                    and
                    <i>becn1</i>
                    transcripts), we queried public gene expression data repositories (<a title="Link to external resource" href="http://www.oncomine.org/">http://www.oncomine.org/</a>) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0046">Rhodes et al., 2004</a>). In<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>B, we reported that
                    <i>atg12</i>
                    and
                    <i>becn1</i>
                    transcripts increased substantially in three separate biological clones of MDA‐MB‐468 breast cancer
                    cells (carrying mutp53R273H) where the mutant p53 knock‐down was induced by doxycycline (DOX) [our
                    elaboration of the data set from Freed‐Pastor and colleagues published in (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0020">Freed‐Pastor et al., 2012</a>)]. This result was further
                    corroborated by our additional analysis of the data set from Gluck et al. (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0027">Gluck et al., 2012</a>). where
                    <i>atg12</i>
                    and
                    <i>becn1</i>
                    transcripts were significantly down‐regulated in breast cancer patients carrying mutant p53,
                    compared to those with WTp53 (<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>C). Altogether
                    these data strongly confirm the inhibitory role of mutant p53
                    <i>in vitro</i>
                    and
                    <i>in vivo</i>
                    on the expression level of ATGs. Moreover, to test the hypothetical prognostic value of ATGs in
                    cancer patients, we considered a gene expression signature (<i>atg12</i>,<i>becn1</i>,<i>sesn1</i>,
                    and<i>dram1</i>) by selecting autophagy‐related genes that were up‐regulated by mutant p53 depletion
                    in<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>A. About 70% of the breast cancer
                    alterations in
                    <i>TP53</i>
                    are missense mutations (TCGA) and where only missense mutations compared to the frame‐shift and
                    non‐sense mutations have been demonstrated to have Gain‐Of‐Function activities (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0002">Bertheau et al., 2013</a>). It is for this reason why we
                    focused our attention only on a subgroup of breast cancer patients expressing mutant p53 proteins
                    independent by intrinsic subtype (luminal, basal‐like, Her+) to assess the potential correlation
                    between the experimental autophagic gene signature and the patient's clinical outcome (<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>D). We analyzed gene expression datasets and
                    survival information downloaded from GEO and sorted for the presence of
                    <i>TP53</i>
                    gene mutations (<a title="Link to external resource" href="http://www.ncbi.nlm.nih.gov/geo/">www.ncbi.nlm.nih.gov/geo/</a>) (<a title="Link to external resource" href="http://www.kmplot.com">www.kmplot.com</a>) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0031">
                        Gyorffy et al., 2010</a>). To analyze the prognostic value of the signature, we split
                    patients into two groups (high and low expression levels of autophagic gene signature) by the median
                    of the expression values. The two groups were then compared in terms of relapse free survival (RFS)
                    and distant metastasis free survival (DMFS). KM analysis in the<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>D showed that low<i>atg12</i>,<i>dram1</i>,
                    <i>sesn1</i>
                    and
                    <i>becn1</i>
                    signature expression was significantly associated with a poor prognosis in the mutant p53 breast
                    cancer subgroup (RFS Statistically significant results with p‐value = 0.02, N = 188; DMFS
                    Statistically significant results with p‐value = 0.0028, N = 83).
                </p>
                <section class="article-section__inline-figure"><figure class="figure" id="mol220161071008-fig-0003"><img class="figure__image" src="https://placeholdit.imgix.net/~text?txtsize=33&amp;txt=TestImage&amp;w=500&amp;h=500"></img><figcaption class="figure__caption"><strong>Figure
                3</strong> 
                        Autophagy‐related gene regulation by mutant p53. (A) The indicated cell lines were transfected
                        with si‐p53 and si‐GFP (as control) oligonucleotides. H1299 cells were transfected with
                        mutp53R175H, mutp53R273H or pcDNA3 (mock) vectors for 48 h. Gene expression analysis of the
                        autophagy‐related genes ATG12, BECN1, SESN1, SESN2, and DRAM1 was performed by RT‐qPCR and was
                        normalized to GAPDH mRNA. *p &lt; 0.05 si‐p53 vs si‐GFP; §p &lt; 0.05 R175H or R273H vs mock.
                        All the experiments are representative of three biological replicates. P‐values were calculated
                        with two‐tailed t‐test. (B) Box plot of ATG12 and BECN1 mRNAs expression obtained in MDA‐MB‐468
                        breast cancer cells by elaboration of data set from Freed‐Pastor and colleagues (Freed‐Pastor
                        et al., 2012; Rhodes et al., 2004). (C) Box plots of ATG12 and BECN1 mRNAs expression in breast
                        cancer patients (n = 72 for wt‐p53 and n = 72 for mut‐p53). Data from Gluck and colleagues were
                        obtained from www.oncomine.org website (Gluck et al., 2012). (D) Kaplan‐Meier survival curves of
                        relapse free survival (RFS) and distant metastasis free survival (DMFS) of breast cancer
                        patients bearing mutant TP53 gene classified according to the expression of atg12, becn1, sesn1,
                        and dram1 signature. RFS Statistically significant results with p‐value = 0.02, N = 188; DMFS
                        Statistically significant results with p‐value = 0.0028, N = 83). The two compared groups are
                        the patients with the highest expression (red) levels of the signature versus the patients with
                        the lowest expression (black).
                    </figcaption></figure></section>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec3.4">
                
                <h3 class="article-section__sub-title">3.4 Mutant p53 inhibits atg12 expression by interaction with NF‐κB p50 subunit</h3>
                <p>We and others have shown that mutant p53 binds to the promoters of its target genes through
                    cooperating with other transcription factors, such as E2F1, NF‐Y, Sp1 and others (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0012 #mol220161071008-bib-0052 #mol220161071008-bib-0053 #mol220161071008-bib-0056 #mol220161071008-bib-0061">
                        Di Agostino et al., 2006; Stambolsky et al., 2010; Strano et al., 2007; Valenti et al., 2015;
                        Weisz et al., 2007</a>). Recently, we have also shown that mutant p53 was able to repress the
                    DNA repair gene expression by the association with the transcriptional repressor E2F4 (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0056">Valenti et al., 2015</a>). Since ATG12 is an essential
                    mediator of the initial phases of autophagosome vesicle formation, we studied this gene looking for
                    promotorial consensus regions binding transcriptional repressors. Our
                    <i>in silico</i>
                    analysis by using MatInspector software (<a title="Link to external resource" href="http://www.genomatix.de">www.genomatix.de</a>)
                    revealed the presence of two NF‐κB p50 specific consensus sequences in the
                    <i>atg12</i>
                    promoter:
                    <i>atg12</i>
                    promoter Region 1 (−1055 bp) and
                    <i>atg12</i>
                    promoter Region 2 (−513 bp) (<a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>A). MatInspector
                    software uses TRANSFAC matrix that was able to discriminate between the specificity of binding
                    between NF‐κB p50 and NF‐κB p65 factors conferring different and specific position frequency to the
                    nucleotides belonging to the promotorial consensus sequence (<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 7</a>). As NF‐κB p50 usually
                    acts as a transcriptional repressor (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0016 #mol220161071008-bib-0054">Elsharkawy et al., 2010; Tong
                        et al., 2004</a>), we studied its recruitment onto the
                    <i>atg12</i>
                    promoter. Chromatin immunoprecipitation experiments (ChIP) in SKBr3 cells revealed that p50 was
                    actually bound to both two binding regions of
                    <i>atg12</i>
                    promoter and that this protein‐DNA interaction was removed by transfection of siRNAp50 oligos (<a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>B;<a class="suppLink" href="#mol220161071008-sup-appsec1">
                        Supplementary Figure 1A</a>). Notably, ChIP assay showed that also mutant p53 protein was
                    recruited onto p50 consensus sites and its binding decreased after p50 depletion (<a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>B). As expected by the
                    <i>in silico</i>
                    analysis, NF‐κB p65 did not bind the two atg12 promoter regions instead recruited NF‐κB p50 (<a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>B). The same results were observed in the
                    ChIP experiment in Panc1 cells (<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary
                        Figure 1B</a>). In order to support new evidence that mutant p53 and p50 bind onto the
                    <i>atg12</i>
                    gene promoter, we aimed to investigate the existence of a novel floating mutp53/p50 protein complex.
                    Coimmunoprecipitation experiments from whole protein extracts of Panc1 and SKBr3 cancer cell lines
                    endogenously expressing p53 mutant proteins revealed the formation of a mutp53/p50 protein complex (
                    <a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>C). NF‐κB p65 was not detected in this protein
                    complex in any coimmunoprecipitation conditions, confirming its absence on the atg12 promoter
                    previously discussed (<a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>C). The formation of
                    endogenous mutant p53/p50 protein complex was further observed also in MDA‐MB‐468 and MDA‐MB‐231
                    cancer cells (<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 8</a>). We supported
                    this result by also showing the formation of the complex between the overexpressed mutp53R175H and
                    endogenous p50 proteins in H1299 cells (<a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>C).
                    Strikingly, the depletion of p50 led to the resumption of
                    <i>atg12</i>
                    mRNA transcription in mutant p53 Panc1 and MDA‐MB‐468 cells (<a class="figureLink" href="#mol220161071008-fig-0004">
                        Figure 4</a>D). According to the above results, the p50 knock down had a functional effect on
                    the increase of autophagosome formation only in SKBr3 and Panc1 cells (bearing mutant p53), thus
                    supporting the inhibitory role of mutant p53/p50 complex in the autophagic process (<a class="figureLink" href="#mol220161071008-fig-0004">Figure 4</a>E).
                </p>
                <section class="article-section__inline-figure"><figure class="figure" id="mol220161071008-fig-0004"><img class="figure__image" src="https://placeholdit.imgix.net/~text?txtsize=33&amp;txt=TestImage&amp;w=500&amp;h=500"></img><figcaption class="figure__caption"><strong>Figure
                4</strong> Mutant p53 inhibits atg12 expression by interaction with NF‐κB p50 subunit. (A) Schematic
                        representation of ATG12 promoter gene regions containing NF‐κB p50 consensus box sequences
                        analyzed in ChIP assays. ATG12 promoter: two regions at 1055 bp (region 1) and 513 bp (region 2)
                        upstream the first exon of ATG12 gene. The TSS is indicated and it was predicted by Eponine
                        software (Down and Hubbard, 2002). (B) Cross‐linked chromatin derived from sip50‐treated SKBr3
                        cells and from siGFP‐transfected control, was immunoprecipitated with the indicated antibodies
                        or in the absence of antibody, and analyzed by RT‐qPCR with specific primers (see Material and
                        Methods or Supplementary Table 2) for the indicated regions in the Figure 4A. Each point of the
                        experiment was carried out in biological duplicate and each reaction of each sample in RT‐qPCR
                        was carried out in technical replicate. The P‐values were calculated with two‐tailed t‐test. *p
                        &lt; 0.05. (C) (Left panel) Immunoprecipitations of NF‐κB p50 or NF‐κB p65 proteins and western
                        blot analysis for p53 binding are performed from lysates of the indicated cancer cell lines
                        expressing mutant p53 proteins (described in Material and Methods). NF‐κB p50 or NF‐κB p65
                        proteins were immunoprecipitated with their relative rabbit polyclonal antibody and the same
                        amount of rabbit IgG was used as negative control of IP. (Right panel) Cell lysate from H1299
                        cells over‐expressing mutp53R175H protein was immunoprecipitated with p53 sheep polyclonal
                        antibody. Cells transfected with the empty vector (pcDNA3) was used as negative control. (D)
                        Cells were transfected with the siRNAp50 or its negative control for 48 h. Gene expression
                        analysis of ATG12 mRNA was performed by RT‐qPCR and normalized to GAPDH mRNA. This experiment
                        was a biological triplicate. The P‐values were calculated with two‐tailed t‐test. Statistically
                        significant results were with p‐value&lt;0.05. (E) Cells were transfected with siRNAp50 or its
                        relative negative control. Autophagosome formation assay was performed using the incorporation
                        of MDC probe. Statistical analysis: *p &lt; 0.05 p53 or p50 vs No Ab; § sip50 vs CTRL
                    </figcaption></figure></section>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec3.5">
                
                <h3 class="article-section__sub-title">3.5 Mutant p53 proteins inhibit AMPK and stimulate mTOR signaling</h3>
                <p>In order to examine whether mutant p53 impinged in the crucial regulatory kinase complexes of the
                    autophagic process we analyzed the protein expression and the phosphorylation state in different
                    cancer cell lines expressing wild type or p53 mutant proteins. We found that the knock down of
                    mutant p53 in either Panc1 and SKBr3 cell lines led to an increase of Sestrin1 and Sestrin2 protein
                    levels, while silencing of wild type p53 in PaCa3 and MCF7 cancer cells resulted in a general
                    down‐regulation of Sestrin protein levels (<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>A).
                    Sestrins are a class of proteins that can directly interact with AMPK subunits favoring their
                    phosphorylation by upstream kinases and thereby resulting in AMPK signaling stimulation (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0005 #mol220161071008-bib-0042 #mol220161071008-bib-0048">Budanov
                        and Karin, 2008; Morrison et al., 2015; Sanli et al., 2012</a>). To determine whether mutant
                    p53 could affect the phosphorylation of AMPK, we analyzed the level of phospho‐AMPKα (Thr172) after
                    modulation of mutant or wild type p53 expression. We observed that depletion of mutant p53 was able
                    to increase the phosphorylation level of AMPK [P(Thr172)‐AMPK/AMPK] and to decrease that of p70S6K
                    [P(Thr)389‐p70S6K/p70S6K], a direct target of mTOR signaling (<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>A). Similar results were obtained with
                    pLVTHM‐p53 vector confirming data observed with pRSUPER‐p53 vector (<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 9</a>). Accordingly, the ectopic
                    expression of mutp53R175H or mutp53R273H clearly determined the repression of AMPK phosphorylation
                    and the increase of p70S6K phosphorylation (<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>
                    A). To better represent our results, we quantified P‐AMPK/P‐p70S6K ratio (<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>B), highlighting that wild type and mutant
                    p53 proteins produced inverse effects on these signaling pathways. Overall, quantification and
                    statistical evaluation of the proteins representatively shown in<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>A are reported in<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 10</a>. To address the role of
                    AMPK phosphorylation mutant p53‐dependent, we inhibited AMPK activity treating the cancer cells with
                    compound C (CC) or overexpressing a dominant negative (DN) AMPK isoform (<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>C and D). Strikingly, AMPK signaling
                    inhibition by both CC and DN‐AMPK strongly rescued the autophagosome formation induced by mutant p53
                    knock down (<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>C). Consistently, in the same
                    conditions of AMPK signaling inhibition the oncogenic proliferation impaired by mutant p53 knock
                    down was restored in all three cell lines tested (<a class="figureLink" href="#mol220161071008-fig-0005">
                        Figure 5</a>D). Overall, these results provide robust evidence that mutant p53 exerts its
                    inhibitory activity on the autophagic signaling throughout the constitutive blockage of AMPK
                    signaling.
                </p>
                <section class="article-section__inline-figure"><figure class="figure" id="mol220161071008-fig-0005"><img class="figure__image" src="https://placeholdit.imgix.net/~text?txtsize=33&amp;txt=TestImage&amp;w=500&amp;h=500"></img><figcaption class="figure__caption"><strong>Figure
                5</strong> 
                        Mutant p53 inhibits AMPK and stimulates mTOR. (A) Cells were seeded in 100‐mm diameter culture
                        dishes and transfected for 48 h with the pRSuper‐p53 vector (shp53) and with plasmids for the
                        ectopic expression of mutant p53 (o.e. R175H; o.e. R273H), or their relative negative control
                        (shCTRL or mock, respectively). Western blotting analysis from these cells was performed with
                        50 μg of whole cell extracts and probed with the indicated antibodies. This analysis was
                        representative of three biological replicates. (B) Quantitative analysis of P‐AMPK/P‐p70S6K
                        ratio. The bands shown in (A) were scanned as digital peaks and the areas of the peaks were
                        reported as fold change, as described in Material and Methods. (C and D) The indicated cell
                        lines were transfected for 48 h with the pRSuper‐p53 vector (shp53) or its negative control
                        (shCTRL), in the absence or presence of 20 μM Compound C (CC) or together the expression vector
                        coding for WT or dominant negative (DN; mutant R531G) AMPK γ2 subunit. Autophagosome formation
                        (C) and cell growth (D) were analyzed by using MDC and crystal violet staining, respectively.
                        Statistical analysis: *p &lt; 0.05 shp53 vs CTRL; ϕ R175H or R273H vs mock; §p &lt; 0.05
                        shp53+CC vs shp53; #p &lt; 0.05 shp53+DN‐AMPK vs shp53+WT‐AMPK.
                    </figcaption></figure></section>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec3.6">
                
                <h3 class="article-section__sub-title">3.6 Mutant p53 inhibits Beclin1 phosphorylation by mTOR stimulation and its expression
                    level
                </h3>
                <p>Given that mTOR pathway was controlled by mutant p53 (<a class="figureLink" href="#mol220161071008-fig-0005">
                    Figure 5</a>), we speculated that our finding was in correlation with the fact that mTOR complex
                    might de‐phosphorylate Beclin1 via ULK1 inhibition impairing the formation of Beclin1‐mediated
                    autophagic protein complexes and autophagy maturation (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0047">
                        Russell et al., 2013</a>). To gain insight into the molecular mechanisms controlling the
                    Beclin1 expression, we observed that the overexpression of mutp53R175H or mutp53R273H proteins in
                    AsPC1 cells decreased both Beclin1 phosphorylation in Serine 15 and its total protein expression
                    level (<a class="figureLink" href="#mol220161071008-fig-0006">Figure 6</a>A). Accordingly, the depletion of
                    mutant p53 from Panc1 cells exerted the inverse results (<a class="figureLink" href="#mol220161071008-fig-0006">
                        Figure 6</a>A). These data correlated with the Beclin1 mRNA expression shown in<a class="figureLink" href="#mol220161071008-fig-0003">Figure 3</a>.
                </p>
                <section class="article-section__inline-figure"><figure class="figure" id="mol220161071008-fig-0006"><img class="figure__image" src="https://placeholdit.imgix.net/~text?txtsize=33&amp;txt=TestImage&amp;w=500&amp;h=500"></img><figcaption class="figure__caption"><strong>Figure
                6</strong> 
                        Mutant p53 inhibits Beclin1. (A) AsPC1 cells were seeded in 100‐mm diameter culture dishes and
                        transfected for 48 h with the plasmids coding for mutant p53 proteins (R175H or R273H), or with
                        pCDNA3 as negative control (mock), in the absence or presence of 5 μM everolimus (EVE). Panc1
                        cells were knocked‐down for mutant p53 expression by 48 h of pRSuper‐p53 vector transfection
                        (shp53). Whole‐cell extracts were used for Western blot analysis using the indicated antibodies.
                        (B and C) Panc1 and SkBr3 cells were seeded in 96‐well plates and transfected with the
                        pRSuper‐p53 vector (shp53) and/or si‐Beclin1 oligos (siBeclin1). Autophagosome formation (B) and
                        cell growth (C) were analyzed after 48 h by the transfection using MDC and crystal violet
                        staining, respectively. These experiments were biological triplicates. The P‐values were
                        calculated with two‐tailed t‐test. Statistical analysis: §p &lt; 0.05 shp53+siBeclin1 vs shp53.
                    </figcaption></figure></section>
                <p>In line with the findings from Russel and colleagues, we also investigated whether the stimulation of
                    mTOR signaling by mutant p53 (shown in<a class="figureLink" href="#mol220161071008-fig-0005">Figure 5</a>) may be
                    responsible for the decrease in levels of Beclin1 phosphorylation. We observed that the addition of
                    everolimus (RAD001), an inhibitor of mTOR currently used in the therapy of several human cancers,
                    was able to completely restore Beclin1 phosphorylation repressed by overexpression of both R175H or
                    R273H p53 mutant proteins, indicating that mTOR stimulation by mutant p53 is highly involved in
                    Beclin1 de‐phosphorylation (<a class="figureLink" href="#mol220161071008-fig-0006">Figure 6</a>A). Quantification
                    and statistical evaluation of Beclin1 and of its phosphorylated form are shown in<a class="suppLink" href="#mol220161071008-sup-appsec1">Supplementary Figure 11</a>. Since Beclin1
                    phosphorylation has been described to be crucial for ATG proteins interaction during autophagosome
                    vesicles formation, we investigated the role of Beclin1 repression on overall autophagy inhibition
                    by mutant p53. We observed that the knockdown of Beclin1 (<a class="suppLink" href="#mol220161071008-sup-appsec1">
                        Supplementary Figure 12</a>) significantly reduced the autophagosome formation induced by the
                    depletion of mutant p53 protein (<a class="figureLink" href="#mol220161071008-fig-0006">Figure 6</a>B), thus
                    suggesting a role for Beclin1 repression on the autophagy pathway inhibition driven by mutant p53.
                    According to this, the concomitant depletion of mutant p53 and Beclin1 led to a partial rescue of
                    proliferation ability of cancer cells as compared to mutant p53 knockdown conditions (<a class="figureLink" href="#mol220161071008-fig-0006">Figure 6</a>C). Overall, our data strongly support the
                    negative effect of mutant p53 on the Beclin1 functionality throughout the inhibition of its
                    expression and phosphorylation via mTOR stimulation.
                </p>
            </div>
            <div class="article-section__sub-content" id="mol220161071008-sec-sec3.7">
                
                <h3 class="article-section__sub-title">3.7 Mutant p53 sensitizes cancer cells to mTOR inhibition</h3>
                <p>Everolimus is an inhibitor of mammalian target of rapamycin (mTOR) routinely used in the targeted
                    therapy of many cancers. We aimed to investigate whether cancer cells bearing mutant p53 might be
                    more responsive to everolimus than wild type p53 cancer cells, accordingly with the inhibitory role
                    of mutant p53 on autophagy via mTOR pathway induction (<a class="figureLink" href="#mol220161071008-fig-0005 #mol220161071008-fig-0006">Figures 5 and 6</a>). To this
                    end, we incubated everolimus Panc1, SkBr3 and MCF7 cell lines and observed that the cells expressing
                    mutant p53 (Panc1 and SKBr3) were more sensitive than cells with wild type p53 (MCF7) to the
                    treatment as reported by the IC
                    <sub>25</sub>
                    and IC
                    <sub>50</sub>
                    values (<a class="figureLink" href="#mol220161071008-fig-0007">Figure 7</a>A). To functionally demonstrate the
                    involvement of mutant p53 on cancer cell sensitivity to everolimus, we evaluated the response of the
                    cells after knockdown or overexpression of p53 mutant proteins. Our data reported in<a class="figureLink" href="#mol220161071008-fig-0007">Figure 7</a>B, showed that mutant p53 knockdown
                    significantly decreased Panc1 and SkBr3 cell response to everolimus treatment if compared to
                    controls. To corroborate this result, we assessed that the overexpression of R175H or R273H mutant
                    p53 conferred to p53‐null AsPC1 cells a strong sensitization to everolimus incubation, as compared
                    to its negative control (mock vector). Moreover, the dominant negative effect of p53 mutant proteins
                    ectopically expressed in WTp53 cells (PaCa3), allowed a stronger impact resulting in a higher
                    response to everolimus incubation in comparison to the PaCa3 cells under control conditions (<a class="figureLink" href="#mol220161071008-fig-0007">Figure 7</a>B). Overall, these results demonstrated that
                    targeting the mTOR pathway may represent an effective therapeutic strategy for cancer patients
                    bearing mutations in the
                    <i>TP53</i>
                    gene.
                </p>
                <section class="article-section__inline-figure"><figure class="figure" id="mol220161071008-fig-0007"><img class="figure__image" src="https://placeholdit.imgix.net/~text?txtsize=33&amp;txt=TestImage&amp;w=500&amp;h=500"></img><figcaption class="figure__caption"><strong>Figure
                7</strong> 
                        Mutant p53 sensitizes cancer cells to everolimus. (A) Cells were seeded in 96‐well plates and
                        treated for 48 h with increasing concentrations of everolimus. Cell growth was measured using
                        the crystal violet colorimetric assay and IC25 and IC50 values for each cell line tested have
                        been reported. (B) Cells were seeded in 96‐well plates and transfected with pRSuper‐p53 vector
                        (shp53) and with the plasmids coding for mutant p53 proteins (R175H or R273H), or with empty
                        pRSuper (shCTRL) or pCDNA3 (mock), respectively, as negative controls. For each experimental
                        condition, cells were untreated or treated with 5 μM everolimus for 48 h. Cell growth was
                        determined using the crystal violet colorimetric assay. The rate of cell growth inhibition
                        corresponds to the effect of everolimus, as compared to vehicle (untreated), in each indicated
                        transfection condition. These experiments were at least biological triplicates. The P‐values
                        were calculated with two‐tailed t‐test. Statistical analysis: *p &lt; 0.05 shp53 vs shCTRL; §p &lt; 0.05
                        R175H or R273H vs mock. (C) Model of the molecular mechanism unraveled in the present study.
                    </figcaption></figure></section>
            </div>
        </div>
        <div class="article-section__content" id="mol220161071008-sec-sec4">
            
            <h2 class="article-section__title">4 Discussion</h2>
            <p>Autophagy is a tightly‐regulated catabolic process of cellular self‐digestion by which cellular
                components are targeted to lysosomes for their degradation. However, the role of autophagy in modulating
                cell death is highly dependent on the metabolic context and on the microenvironmental conditions in
                which the cells lie (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0037">Lorin et al., 2013</a>). Key functions of
                autophagy are aimed to provide energy and metabolic precursors under conditions of starvation and to
                alleviate cellular stress removing damaged proteins and organelles, which are deleterious for cell
                survival. Therefore, autophagy appears to serve as a pro‐survival stress response in most settings (
                <a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0019 #mol220161071008-bib-0040">Fiorini et al., 2013; Matsuzawa et al.,
                    2015</a>). Increasing evidence show its pivotal role in cell death after treatment with
                anti‐cancer drugs or under particular stressful conditions (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0017 #mol220161071008-bib-0022">Fiorini et al., 2015a; Fulda and
                    Kögel, 2015</a>). Thus, despite the net effect of autophagy in cancer cells is highly contextual,
                substantial results describe that enforced over‐activation of autophagy can lead to excessive cellular
                self‐digestion via the autophagosomal‐lysosomal pathway determining autophagic cell death (also named
                cell death type II) (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0024">Galluzzi et al., 2012</a>). The
                importance of autophagic cell death becomes evident by the consideration that it is able to bypass drug
                resistance, especially in apoptosis‐refractory tumors, offering new therapeutic options in overcoming
                cancer resistance mechanisms (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0022 #mol220161071008-bib-0050">Fulda and
                    Kögel, 2015; Schonthal, 2009</a>). Since Avantaggiati and colleagues have previously demonstrated
                that mutant p53 stability is controlled by lysosomal degradation and not by proteasomal involvement as
                occurs for wild type p53 (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0007">Choudhury et al., 2013</a>),
                autophagy inhibition by mutant p53 may, at least partially, explain also the high expression levels of
                mutant p53 itself observed in cancer cells.
            </p>
            <p>Recently Mathew and colleagues have clearly shown how defects in the autophagic pathway could diminish
                survival and enhance tumorigenesis with the concomitant accumulation in the DNA mutations (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0039">Mathew et al., 2007</a>). The high mutation rate and the
                severe genomic instability are the most important features of the cancers expressing mutant p53 proteins
                (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0045 #mol220161071008-bib-0051">Negrini et al., 2010; Soussi and
                    Wiman, 2007</a>). Intriguingly, in this study we intended to deeply investigate the molecular
                mechanisms related to autophagy inhibition by GOF p53 mutants, which can be summarized with the
                following considerations schematically illustrated in<a class="figureLink" href="#mol220161071008-fig-0007">
                    Figure 7</a>C: i) mutant p53 inhibits the expression of a panel of autophagy‐related genes,
                including<i>becn1</i>,<i>dram1</i>,<i>sesn1/2</i>, and<i>atg12</i>; ii) autophagy inhibition by mutant
                p53 is mediated by its interaction with p50 NF‐κB subunit on the
                <i>atg12</i>
                gene promoter; iii) mutant p53 inhibits SESN1/AMPK axis and stimulates mTOR pathway; iv) mutant p53
                inhibits P‐Beclin1 via mTOR sensitizing cancer cells to everolimus.
            </p>
            <p>During the last several years, Morselli et al. stated that the cytoplasmic localization is the
                preponderant requirement for mutant p53 to inhibit autophagy indicating that those variants of mutant
                p53 that preferentially localize in the cytoplasm effectively repress autophagy, whereas p53 mutants
                that display a prominently nuclear distribution fail to inhibit it (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0043">Morselli et al., 2008</a>). However, these previous
                observations exclusively reached by mutant p53 over‐expression assays are not confirmed by our
                investigation. Indeed, we reveal that all tested GOF p53 mutant variants, listed in<a class="tableLink" title="Link to table" href="#mol220161071008-tbl-0001">Table 1</a>, acquire the property to inhibit autophagy
                irrespectively of their tissue of origin, such as breast, pancreas or lung, or of their prevalent
                subcellular localization previously determined by Morselli et al. (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0043">Morselli et al., 2008</a>). In addition, our results
                indicate that the repression of the autophagic machinery is functional to the hyper‐proliferating and
                anti‐apoptotic activities of these hot‐spot p53 mutants in cancer cells. Very recently, Gu et al. have
                identified an autophagy‐related prognostic signature for breast cancer (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0029">Gu et al., 2015</a>) analyzing six independent public‐domain
                microarrays from GEO database. They reported that inactivation of autophagy was associated with
                shortened survival of breast cancer patients and that a set of eight genes (<i>bcl2</i>,<i>birc5</i>,<i>
                    eif4ebp1</i>,<i>ero1L</i>,<i>fos</i>,<i>gapdh</i>,
                <i>itpr1</i>
                and<i>vefgA</i>) were assigned as an autophagy‐related prognostic signature for breast cancer. However,
                despite that their statement is consistent with our data, the genes identified by Gu et al. have a high
                level of functional versatility thus lacking in autophagy specificity of the signature. In this study,
                we show that low expression of<i>atg12</i>,<i>dram1</i>,
                <i>sesn1</i>
                and
                <i>becn1</i>
                autophagy‐related signature was significantly associated with a poor prognosis in the mutant p53 breast
                cancer subgroup. This consideration strongly reinforces the statement of autophagy as an
                anti‐proliferative and pro‐apoptotic event that cancer cells try to escape<i>in vivo</i>. Furthermore,
                autophagy‐deficient cell lines increase DNA double‐strand breaks and other chromosome abnormalities
                suggesting a role for the autophagic pathway in suppressing genome damage and inhibition of
                pro‐tumorigenic process (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0039">Mathew et al., 2007</a>). In
                addition, we demonstrated a protein‐protein interaction between p50 NF‐κB subunit and mutant p53 and
                their recruitment in two regions of
                <i>atg12</i>
                promoter. Intriguingly, we observed that the depletion of p50 protein increased both
                <i>atg12</i>
                mRNA expression and autophagosome formation in cancer cells bearing mutant p53. Therefore, it is
                tempting to speculate that the interaction between mutant p53 and the p50‐p50 homodimer could represent
                a novel molecular mechanism by which mutant p53 is able to repress the expression of its target genes (
                <a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0030 #mol220161071008-bib-0054">Guan et al., 2005; Tong et al.,
                    2004</a>). The existence of this functional interaction might have a great impact in cancer cell
                biology contributing to the discovery of new drugs potentially able to inhibit this oncogenic mechanism.
            </p>
            <p>Concerning signal transduction pathways involved in autophagy inhibition, our data are in accordance with
                a recent study of Myers and colleagues which indicated that under conditions of energy stress, GOF p53
                mutants, but not WTp53, can bind to the AMPKα subunit inhibiting AMPK signaling (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0062">Zhou et al., 2014</a>). We further documented that mutant
                p53 suppressed the expression of Sestrins which have been described to act as AMPK activators through
                the direct interaction with AMPK complex favoring its phosphorylation by upstream kinases (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0005 #mol220161071008-bib-0042 #mol220161071008-bib-0048">Budanov and
                    Karin, 2008; Morrison et al., 2015; Sanli et al., 2012</a>). Overall, we showed that AMPK
                signaling inhibition was functionally implicated on autophagy counteraction by mutant p53. Moreover, we
                demonstrated that mTOR pathway stimulation by mutant p53 was able to repress Beclin‐1 phosphorylation,
                which is required for activating the ATG14L‐VPS34 complex during autophagosome formation (<a class="bibLink" title="Link to bibliographic citation" href="#mol220161071008-bib-0047">Russell et al., 2013</a>), thus orchestrating autophagy
                machinery inhibition driven by mutant p53. Indeed, it is important to highlight that mutant p53
                orchestrated a number of adjustments aimed at inhibiting the autophagic flow starting from ATG
                regulation to autophagic vesiscles formation and their fusion with lysosomes. We finally underline the
                translational relevance of our data concerning the potential use of the mTOR inhibitor everolimus for
                cancer patients carrying mutant
                <i>TP53</i>
                gene.
            </p>
        </div>
        <div class="article-section__content" id="mol220161071008-sec-sec5">
            
            <h2 class="article-section__title">5 Conclusions</h2>
            <p>Overall, we have observed that mutant p53: i) inhibited mRNA and protein expression levels of various key
                ATGs (<a class="figureLink" href="#mol220161071008-fig-0003 #mol220161071008-fig-0004 #mol220161071008-fig-0005 #mol220161071008-fig-0006">
                    Figures 3, 4, 5 and 6</a>); ii) inhibited the formation of autophagic vesicles and formation of
                the mature isoform of LC3 (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>); iii) inhibited
                fusion of the autophagic vesicles with lysosomes, formation of autolysosomes and lysosomal acidification
                of the lumen (<a class="figureLink" href="#mol220161071008-fig-0001">Figure 1</a>); iv) inhibited AMPK signaling and
                stimulated mTOR pathway making cancer cells bearing mutant p53 susceptible to mTOR inhibition (<a class="figureLink" href="#mol220161071008-fig-0005 #mol220161071008-fig-0007">Figures 5 and 7</a>). Altogether
                these observations also provided new prognostic tools and innovative insights into mutant p53 GOF
                activities in cancer cell biology, strongly supporting mTOR inhibition as a personalized therapeutic
                strategy for cancer patients carrying mutant p53.
            </p>
        </div>
        <div class="article-section__content" id="mol220161071008-sec-sec6">
            <h2 class="article-section__title"> Conflict of interest</h2>
            <p>The authors declare that they have no conflicts of interest.</p>
        </div>
        <div class="article-section__content" id="mol220161071008-sec-ack0010">
            <h2 class="article-section__title"> Acknowledgments</h2>
            <p>We wish to thank Tania Merlino for manuscript editing. This work was supported by Italian Association for
                Cancer Research (AIRC)‐Fondazione CariPaRo, Padova, Italy (n. C98C13000210007); Joint Projects program
                2015 from University of Verona to M. Donadelli (n. B12I15002320003); Italian Association for Cancer
                Research (AIRC) to S. Di Agostino (AIRC IG‐n.16984) and to G. Blandino (AIRC IG‐n.14455). M.
                Nadal‐Serrano was supported by a short‐term fellowship for the stay abroad (University of Verona, Italy)
                from European Molecular Biology Organization (EMBO). I. Dando is a fellow of AIRC (AIRC‐Fondazione
                CariPaRo, Padova, Italy) and NanoMedicine (Fondazione Cariverona, Project Verona Nanomedicine
                Initiative). E. Dalla Pozza is a fellow of ARC‐Net (Applied Research on Cancer Network), University of
                Verona, Italy. E. Butturini is a fellow of Joint Project (University of Verona, Italy and Aboca,
                Sansepolcro, Arezzo, Italy).
            </p>
        </div>
        <div class="article-section__sub-content" id="mol220161071008-app-appsec2">
            <h2 class="article-section__title">Supplementary data</h2>
            <div class="article-section__content" id="s910">
                <p>Supplementary data related to this article can be found at<a title="Link to external resource" href=" http://dx.doi.org/10.1016/j.molonc.2016.04.001">
                    http://dx.doi.org/10.1016/j.molonc.2016.04.001</a>.
                </p>
            </div>
        </div>
        
    </section>
<section class="article-section article-section__supporting"><h2 class="article-section__title">
                Supporting Information
            </h2><div class="support-info__table-wrapper article-table-content-wrapper"><table class="support-info__table table article-section__table"><thead><th id="article-filename" scope="col">Filename</th><th id="article-description" scope="col">Description</th></thead><tbody><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs1.jpg">supinfo/mol220161071008-sup-figs1.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 1 (A) Cells were seeded in 100‐mm diameter culture dishes and transfected for
                        48 h with sip50 oligos or with negative control (siGFP). 40 μg of total protein extracts were
                        analyzed by Western blotting using p50 NF‐κB and GAPDH or H3 (as normalizing factor) antibodies.
                        (B) Cross‐linked chromatin derived from sip50‐treated Panc1 cells and from siGFP‐transfected
                        control, was immunoprecipitated with the indicated antibodies or in the absence of antibody, and
                        analyzed by RT‐qPCR with specific primers (see Material and Methods or Supplementary Table 2)
                        for the regions of ATG12 promoter indicated in the Figure 4A or in H1H2B2 genic region resulted
                        negative for the recruitment of mutant p53. *p &lt; 0.05.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs2.jpg">supinfo/mol220161071008-sup-figs2.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 2 (A) Panc1 cells were pre‐treated with 1 mM 3MA for 1 h before cell
                        transfection with plasmids coding for LC3‐GFP and pRSuper‐p53 vector (shp53) or empty vector
                        (shCTRL) for 48 h. Cells were fixed and nuclei were stained with Hoescht (blue). Merge images
                        come from a single z‐plane. Scale bar 10 μm. (B) Panc1 cells were pre‐treated with 1 mM 3MA for
                        1 h before cell transfection for 48 h. Whole‐cell extracts were used for Western blot analysis
                        of the autophagic protein LC3 (isoforms I and II), p53 and GAPDH (as control loading). (C‐E)
                        Panc1 and MDA‐MB‐468 cells were seeded in 96‐well plates and pre‐treated with 1 mM 3MA for 1 h
                        before cell transfection for 48 h. Autophagosome formation (C), cell growth (D), and apoptosis
                        (E) were determined using MDC assay, crystal violet colorimetric assay and annexinV/FITC binding
                        assay, respectively. All the experiments presented in this figure are representative of three
                        biological replicates. P‐values were calculated with two‐tailed t‐test. Statistical analysis:
                        *p &lt; 0.05 shp53 vs shCTRL; §p &lt; 0.05 shp53+3MA vs shp53.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs3.jpg">supinfo/mol220161071008-sup-figs3.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 3 Cells were seeded in 96‐well plates and transfected with pRSuper‐p53 vector
                        (shp53), with pCDNA‐p53R175H, pCDNA‐p53R273H plasmids or their negative controls (empty pRSuper
                        and pCDNA3 mock vector, respectively). Cell growth was determined using the crystal violet
                        colorimetric assay. Statistical analysis: *p &lt; 0.05 shp53 vs CTRL; §p &lt; 0.05 R175H or
                        R273H vs mock.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs4.jpg">supinfo/mol220161071008-sup-figs4.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 4 Panc1 cells were transfected with pMSCV‐Puro‐miR30‐shATG5 vector (or its
                        negative empty vector). Gene expression analysis of ATG5 was performed by RT‐qPCR and was
                        normalized to GAPDH mRNA. *p &lt; 0.05 shATG5 vs shCTRL.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs5.jpg">supinfo/mol220161071008-sup-figs5.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 5 Western blot of p53, and GAPDH as normalizing factor, performed in all cell
                        lines used for RT‐qPCR shown in Figure 3A. To exclude back‐side effects of shp53 sequence
                        (pRSuper‐p53 vector) and to confirm the robustness of the data, a commercial siRNA smart pool of
                        three oligonucleotides (si‐p53) transiently targeting p53 (Santa Cruz Biotech. sc‐29435), and
                        its si‐GFP negative control, were used in these experiments.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs6.jpg">supinfo/mol220161071008-sup-figs6.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 6 (A and B) Indicated cell lines were transfected with pRSuper‐p53 vector
                        (shp53), with pCDNA‐p53R175H, pCDNA‐p53R273H plasmids or their negative controls (empty pRSuper
                        and pCDNA3 mock vector, respectively). Gene expression analysis of CCNB1 was performed by
                        RT‐qPCR and was normalized to GAPDH mRNA. *p &lt; 0.05 sip53 vs siGFP; R175H or R273H vs vector.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs7.jpg">supinfo/mol220161071008-sup-figs7.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 7 TRANSFAC matrix of NF‐κB p50 and NF‐κB p65 consensus sequences used by
                        MatInspector software.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs8.jpg">supinfo/mol220161071008-sup-figs8.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 8 Immunoprecipitations of NF‐κB p50 and western blot analysis for p53 binding
                        are performed from lysates of the indicated cancer cell lines expressing mutant p53 proteins, as
                        described in Material and Methods.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs9.jpg">supinfo/mol220161071008-sup-figs9.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 9 Panc1 cells were transfected with pLVTHM‐p53 vector (shp53) or its negative
                        control pLVTHM (shCTRL) to confirm results obtained with pRSUPER‐p53 vector. (A) Autophagosome
                        formation assay was performed using the incorporation of MDC probe. *p &lt; 0.05 shp53 vs shCTRL
                        (B) Western blot of P‐AMPK, AMPK, P‐p70S6K, p70S6Kp53 and GAPDH was performed as described in
                        Material and Methods.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs10.jpg">supinfo/mol220161071008-sup-figs10.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 10 Quantitative analysis of SESN1/GAPDH, SESN2/GAPDH, P‐AMPK/AMPK,
                        P‐p70S6K/p70S6K and p53/GAPDH ratios representatively shown in Figure 5A. The Western blot bands
                        were scanned as digital peaks and the areas of the peaks were reported as fold change, as
                        described in Material and Methods. *p &lt; 0.05 shp53 vs shCTRL; §p &lt; 0.05 R175H or R273H vs
                        mock.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs11.jpg">supinfo/mol220161071008-sup-figs11.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 11 Quantitative analysis of P‐Beclin1, Becin1 and p53 normalized on GAPDH
                        expression representatively shown in Figure 6A. The Western blot bands were scanned as digital
                        peaks and the areas of the peaks were reported as fold change, as described in Material and
                        Methods. §p &lt; 0.05 R175H or R273H vs mock; *p &lt; 0.05 R175H+EVE vs R175H or R273H+EVE vs
                        R273H; # shp53 vs shCTRL.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-figs12.jpg">supinfo/mol220161071008-sup-figs12.jpg</a><br></br>image/jpeg</td><td headers="article-description">Suppl. Figure 12 Cells were seeded in 100‐mm diameter culture dishes and transfected for
                        48 h with siBeclin1 oligos or with negative control (siGFP). 40 μg of total protein extracts
                        were analyzed by Western blot using Beclin1 and GAPDH (as normalizing factor) antibodies.
                    </td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-mmc1.docx">supinfo/mol220161071008-sup-mmc1.docx</a><br></br>application/msword</td><td headers="article-description">Supplementary data</td></tr><tr id="@xml:id"><td headers="article-filename"><a href="supinfo/mol220161071008-sup-mmc2.docx">supinfo/mol220161071008-sup-mmc2.docx</a><br></br>application/msword</td><td headers="article-description">Supplementary data</td></tr></tbody></table></div></section></article></div></div></body></html>
`
